Dockerfile

```diff
@@ -232,3 +232,6 @@ RUN printf "[safe]\n	directory = /src"  > ~semgrep/.gitconfig && \
 # to interactively explore the docker image.
 CMD ["semgrep", "--help"]
 LABEL maintainer="support@semgrep.com"
+
```

Comment:
maybe update the comment 10 lines above about the commented #USER semgrep. Maybe something like "see the rootless docker target below"

Reply:
Done